src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -36,27 +44,36 @@ export const MyProfile = () => {
     queryFn: () => fetchPerson(personId),
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_person'), variant: 'error' })),
   });
-  const person = personQuery.data;
 
-  const hasActiveEmployment = filterActiveAffiliations(person?.affiliations).length > 0;
+  const person = personQuery.data;
 
   const firstName = getValueByKey('FirstName', person?.names);
   const lastName = getValueByKey('LastName', person?.names);
   const personPreferredFirstName = getValueByKey('PreferredFirstName', person?.names);
   const personPreferredLastName = getValueByKey('PreferredLastName', person?.names);
-  const [editPreferredNames, setEditPreferredNames] = useState(false);
   const personTelephone = person?.contactDetails?.telephone;
+  const [editPreferredNames, setEditPreferredNames] = useState(false);
 
   const initialValues: CristinPersonFormData = {
     preferredFirstName: personPreferredFirstName ? personPreferredFirstName : firstName,
     preferredLastName: personPreferredLastName ? personPreferredLastName : lastName,
+    contactDetails: {
+      telephone: personTelephone ? personTelephone : '',
```

Comment:
Litt kunstig å ha en egen const for `telephone` men ikke `email` eller `webPage`. Ingen av disse brukes noen andre plasser
```suggestion
      telephone: person?.contactDetails?.telephone ?? '',
```